# 파이썬 증권 데이터 분석 
증권과 관련된 개념 및 내용 정리 

#### 연평균 성장률 (CAGR, Compound Annual Growth Rates)
: 복합 연평균 성장률 or 연복리 수익률   
1년 동안 얼마 만큼씩 증가하는지를 나타내는 값으로, 주로 투자 수익률을 표시하는데 사용되지만, 판매수량이나 사용자 증가율 등을 나타낼 때도 쓴다. 

![](https://i2.wp.com/img.tfd.com/mgh/Investopedia/Compound_Annual_Growth_Rate.png)

In [1]:
def getCAGR(first, last, year):
    return (last/first)**(1/year) -1

In [8]:
cagr = getCAGR(65300, 2669000, 20)
print ('Samsung EC CAGR : {0:.2%}'.format(cagr))

Samsung EC CAGR : 20.38%


삼성전자는 1998년 4월 27일 65,300원이던 주가가 액면분할 직전인 2018년 4월 27일 2,669,000원이 되기까지의 20년동안 연평균 성장률 CAGR은 20.38%